platform_umbrella/apps/common_core/lib/common_core/defaults/images.ex

```diff
@@ -5,8 +5,8 @@ defmodule CommonCore.Defaults.Images do
 
   @batteries_included_base "#{CommonCore.Version.version()}-#{CommonCore.Version.hash()}"
 
-  @cert_manager_allowed_tags ~w(v1.15.1)
-  @cert_manager_default_tag "v1.15.1"
+  @cert_manager_allowed_tags ~w(v1.15.4)
```

Comment:
Should we just be adding `.4` to the list of allowed tags instead of just replacing `.1`?

Reply:
<img src="https://media0.giphy.com/media/1lTRvRgRI1Y6iKsvI0/giphy-downsized-medium.gif"/> Nice catch. Thanks!